composer.json

```diff
@@ -42,13 +42,11 @@
         }
     },
     "scripts": {
-        "phpstan": "@php ./vendor/bin/phpstan analyse src tests -l 7 -c extension.neon",
-        "psalm": "@php ./vendor/bin/psalm --find-dead-code --threads=8 --diff --diff-methods",
+        "psalm": "@php ./vendor/bin/psalm --find-dead-code --threads=8 --diff",
         "cs-fix": "@php ./vendor/bin/php-cs-fixer fix",
         "test": "@php ./vendor/bin/phpunit"
     },
     "scripts-descriptions": {
-        "phpstan": "Execute PHPStan on PHP7.0+, you need to install it",
```

Comment:
What are the reasons to remove PHPStan from this project?

Reply:
The lib is not in dev dependencies, so the composer can't work
And I tried to add it but I got an error because it's not compatible with PHP5.6 (but our library must be..)